spec/rubocop/cop/performance/string_replacement_spec.rb

```diff
@@ -381,19 +381,25 @@
       it 'when the pattern contains an escape character' do
         new_source = autocorrect_source(cop, "'abc'.gsub('\n', ',')")
 
-        expect(new_source).to eq("'abc'.tr(\"\\n\", ',')")
+        expect(new_source).to eq("'abc'.tr('\n', ',')")
+      end
+
+      it 'when the pattern contains double backslash' do
```

Comment:
Btw, the descriptions of all examples here sound pretty strange. They should read like sentences (with `it` being the start of the sentence). I'd suggest tweaking them in a an extra commit. See http://betterspecs.org.